hushline/static/css/style.css

```diff
@@ -149,7 +149,7 @@ p.instr {
 .badgeContainer {
     display: flex;
     gap: .25rem;
-    margin-bottom: .75rem;
+    margin: .5rem 0;
     align-items: center;
 }
 
```

Comment:
Would also consider 

`.badgeContainer:empty {
    margin: 0;
}`

This pseudo selector applies specific styles if the element doesn't have any child elements.

Reply:
Oh good call. Will add that JIC, but this actually made me reconsider the choice to only show verified badges on the all tab. Since this kind of a special designation, and the tab bar with the Verified label can be scrolled out of the viewport, maybe it's best to include them.